config.template/faf-unitdb/datafiles.json

```diff
@@ -0,0 +1,4 @@
+[
+	"DATA/GAMEDATA/units.scd.3599",
+	"DATA/GAMEDATA/projectiles.scd.3599"
+]
```

Comment:
Seeing that there are version numbers in the files, will this work out of the box when I clone faf-stack, and copy this config over? Or will the application throw a file not found error at me, cause me time to investigate the issue only to find out that the default is misconfigured?

If so, I suggest commenting-out this (at least the content of the array) so it's clear that it's just for reference. Not sure how the application behaves when we do.